os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -126,6 +131,17 @@ static void rtl8730e_lcd_init(void)
 	sem_init(&g_next_frame_block, 0, 1);
 }
 
+static void rtl8730e_lcd_power_off(void)
+{
+	InterruptDis(lcdc_irq_info.num);
+	GPIO_WriteBit(MIPI_GPIO_RESET_PIN, PIN_LOW);
+}
+static void rtl8730e_lcd_power_on(void)
+{
+	GPIO_WriteBit(MIPI_GPIO_RESET_PIN, PIN_HIGH);
+	DelayMs(120);
+	amebasmart_mipi_dsi_host_reinitialize();
```

Comment:
Hi, currently, the resume flow is handled in the wakeup hook. However, since this flow duplicates rtk_mipi_resume and is controlled by the application layer through power_on and backlight, it would be better to remove the following registration from amebasmart_mipi_dsi_host_initialize: pmu_register_sleep_callback(PMU_MIPI_DEVICE, (PSM_HOOK_FUN)rtk_mipi_suspend, NULL, (PSM_HOOK_FUN)rtk_mipi_resume, NULL);

Reply:
Could you please raise new PR about this after this PR merge?

Reply:
Sure, I will do it when this PR is merged.